ops/services/alerts/app_service_metrics/main.tf

```diff
@@ -2,7 +2,8 @@ locals {
   env_title = title(var.env)
 
   // If skip_on_weekends true, only run the query on weekdays
-  skip_on_weekends = var.skip_on_weekends ? "| where dayofweek(now()) between (time(1) .. time(5))" : "| where true"
+  skip_on_weekends    = var.skip_on_weekends ? "| where dayofweek(now()) between (time(1) .. time(5))" : "| where true"
+  skip_when_off_hours = var.skip_when_off_hours ? "| where hourofday(now()) between (time(14) .. time(22))" : "| where true"
 }
```

Comment:
As a note, the above code has been added to give us a variable we can toggle for other alerts on a per-environment basis. Since the requested changes for this one are on `prod`, we're making them inline with the alert query, since these variables traditionally do not apply at that environment level.

Reply:
so skip_when_off_hours is currently not used anywhere, but there are plans to use it on lower envs?

Reply:
No explicit plans as of yet, but we can certainly make them! This can be used in tandem with `skip_on_weekends`, which is available on a per-environment basis.